docs/UserGuide.md

```diff
@@ -204,6 +204,21 @@ Examples:
 * `filter vendor customer` returns all contacts tagged by the tag: vendor and
   all contacts tagged by the tag: customer in JobFestGo.
 
+
+### Adding an event: `add_event`
+
+Adds an event to JobFestGo.
+
+Format: `add_event n/NAME d/DATE a/ADDRESS`
+
+* An event must have mandatory fields name, date and address
```

Comment:
I think it would be better to add a `:` after the fields

Reply:
fixed